packages/client/src/sync/fullsync.ts

```diff
@@ -118,24 +118,19 @@ export class FullSynchronizer extends Synchronizer {
 
   /**
    * Finds the best peer to sync with. We will synchronize to this peer's
-   * blockchain. Returns null if no valid peer is found
+   * blockchain. Returns null if no valid peer is found.
+   *
+   * Note: in pre-merge times selection here was done by TD comparison,
+   * now it is simply a take-first approach (might be valuable to expand/differentiate).
    */
   async best(): Promise<Peer | undefined> {
-    let best
     const peers = this.pool.peers.filter(this.syncable.bind(this))
     if (peers.length < this.config.minPeers && !this.forceSync) return
     for (const peer of peers) {
       if (peer.eth?.status !== undefined) {
-        const td = peer.eth.status.td
-        if (
-          (!best && td >= this.chain.blocks.td) ||
-          (best && best.eth && best.eth.status.td < td)
-        ) {
-          best = peer
-        }
+        return peer
```

Comment:
Ok, this works, I also remember using this as "local hack" to fix it. Will not approve (one more set of eyes first to look at this first) but we should do some research to how other clients filter out good peers vs non-good peers. I think we can merge this as-is though (but will wait for one more set of eyes)